flink/src/main/scala/org/apache/mahout/flinkbindings/blas/FlinkOpAewScalar.scala

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
A elementwise scalar is deprecated now. Better use unaryfunc operator. Then things like A+5 can be represented as 

OpUnaryFunc(A, x=> x+5). 

Unary function mechanism is more generic and affords for unary function fusion optimizations.